source/faq.txt

```diff
@@ -17,6 +17,17 @@ This page provides answers to some frequently asked questions we have
 encountered. If you have additional questions please contact MongoDB
 Support.
 
+Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
```

Comment:
I opted to consolidate that information into one page (the`mongosync` page's behavior section) since it seemed to make more sense there than in the mongosync status reference page. Regardless, in the FAQ section I made some slight adjustments to the wording to make the behavior _during_ sync clearer.